resolwe/flow/views.py

```diff
@@ -281,17 +291,17 @@ def _filter_owner_permission(self, data):
     def _filter_public_permissions(self, data):
         """Raise ``PermissionDenied`` if public permissions are too open"""
 
-        ALLOWED_PUBLIC_PERMISSIONS = ['view', 'add', 'download']
+        allowed_public_permissions = ['view', 'add', 'download']
```

Comment:
:) Probably not. But constants (capitalized) should go to the module namespace. Variables in the function should be small caps.